<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForumMessageView</class>
 <widget class="QWidget" name="ForumMessageView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo View</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="AvatarWidget" name="avatarWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GxsIdLabel" name="nameLabel">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="replyButton">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reply</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/mail/reply.png</normaloff>:/icons/mail/reply.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="RSTextBrowser" name="postText">
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="shareButton">
       <property name="toolTip">
        <string>Copy RetroShare link</string>
       </property>
       <property name="text">
        <string>Share</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/share.png</normaloff>:/images/share.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GxsIdLabel</class>
   <extends>QLabel</extends>
   <header>gui/gxs/GxsIdLabel.h</header>
  </customwidget>
  <customwidget>
   <class>AvatarWidget</class>
   <extends>QWidget</extends>
   <header>gui/common/AvatarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RSTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>gui/common/RSTextBrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
